app/assets/stylesheets/moj/_autocomplete.scss

```diff
@@ -42,7 +47,7 @@
   }
 
   &:hover {
-    cursor: pointer;
+    // cursor: pointer;
```

Comment:
💭 If we're not using this, we might as well get rid of it rather that leaving it commented out.